src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,34 @@
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class Main {
+    public static void main(String[] args) {
+        FileReader fileReader = new FileReaderImpl();
+        List<String> inputReport = fileReader.read("reportToRead.csv");
```

Comment:
The file path 'reportToRead.csv' should be passed as a constant or a variable, preferably placed in the Main class, to avoid hardcoding.